staging/src/k8s.io/apiserver/pkg/authorization/config/validation/validation.go

```diff
@@ -78,30 +76,20 @@ func ValidateAuthorizationConfiguration(fldPath *field.Path, c *authorizationapi
 	return allErrs
 }
 
-func ValidateWebhookConfiguration(fldPath *field.Path, c *authorizationapi.WebhookConfiguration, requireName bool, seenNames sets.String) field.ErrorList {
+func ValidateWebhookConfiguration(fldPath *field.Path, c *authorizationapi.WebhookConfiguration, seenNames sets.String) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if len(c.Name) == 0 {
-		if requireName {
-			allErrs = append(allErrs, field.Required(fldPath.Child("name"), ""))
-		}
+		allErrs = append(allErrs, field.Required(fldPath.Child("name"), ""))
 	} else if seenNames.Has(c.Name) {
 		allErrs = append(allErrs, field.Duplicate(fldPath.Child("name"), c.Name))
-	} else {
-		// TODO: check format? dns label or subdomain?
+	} else if errs := utilvalidation.IsDNS1035Label(c.Name); len(errs) != 0 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("name"), c.Name, fmt.Sprintf("webhook name is invalid: %s", strings.Join(errs, ", "))))
 	}
 	seenNames.Insert(c.Name)
 
-	if c.AuthorizedTTL.Duration == 0 {
-		allErrs = append(allErrs, field.Required(fldPath.Child("authorizedTTL"), ""))
-	}
-
-	if c.UnauthorizedTTL.Duration == 0 {
-		allErrs = append(allErrs, field.Required(fldPath.Child("unauthorizedTTL"), ""))
-	}
```

Comment:
I think we still need the TTL checks in validation (also make sure we can't specify negative durations)